src/homework5/1/runner.py

```diff
@@ -0,0 +1,12 @@
+import inspect
+import pytasks
+
+
+def runner(*args):
+    if not args:
+        func_tuples = inspect.getmembers(pytasks, inspect.isfunction)
```

Comment:
```suggestion
functions = [element for element in pytasks if not element.startswith(__) and callable(element)]
```

Reply:
I changed code to more clear; dir returns string that is not callable, so check is performed later.